catalog-management/src/main/java/com/c12e/cortex/examples/catalog/RailedCommand.java

```diff
@@ -289,14 +216,16 @@ public final void run() {
         SessionExample example = new SessionExample();
         CortexSession cortexSession = example.getCortexSession();
         DocumentContext config;
+        LocalCatalog localCatalog;
         try {
+            localCatalog = new LocalCatalog(specPath);
```

Comment:
Is it not the same thing if you use the `Catalog` implementation from the `CortexSession` ? `Catalog localCatalog = cortexSession.catalog()`? If it is, then it might be preferable since the entrypoint for the user/developer is still the `CortexSession`